scio-macros/src/main/scala/com/spotify/scio/coders/AvroCoderMacros.scala

```diff
@@ -34,7 +34,7 @@ private[coders] object AvroCoderMacros {
       _root_.org.apache.beam.sdk.coders.AvroCoder.of[$companioned](
         classOf[$companioned],
         new $companioned().getSchema,
-        true
+        false
```

Comment:
This should probably stay `true` so current code doesn't change behaviour if relying on internal coders as explained by @clairemcginty in https://github.com/apache/beam/pull/15616 but the reflect API breaks logical type support.